rocketmq-console/src/main/resources/static/view/pages/ops.html

```diff
@@ -16,23 +16,26 @@
   -->
 <div class="container-fluid" id="deployHistoryList">
     <div class="page-content">
-        <h2 class="md-title">NameSvrAddrList</h2>
+        <h2 class="md-title">NameServerAddressList</h2>
         <div class="pull-left">
-            <md-chips ng-model="namesvrAddrList"  readonly="ctrl.readonly" md-removable="ctrl.removable" md-enable-chip-edit="true"></md-chips>
+            <md-chips ng-model="namesvrAddrList" readonly="ctrl.readonly" md-removable="ctrl.removable"
+                      md-enable-chip-edit="true"></md-chips>
         </div>
         <div class="pull-left">
-            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateNameSvrAddr()">{{'UPDATE' | translate}}
+            <button class="btn btn-raised btn-sm btn-primary" type="button" ng-click="updateNameSvrAddr()">{{'UPDATE' |
```

Comment:
Why format here ?

Reply:
Sry,I sumbit another PR to fix it.I will close this.